jedi/evaluate/imports.py

```diff
@@ -271,9 +271,11 @@ def _do_import(self, import_path, sys_path):
 
         module_name = '.'.join(import_parts)
         try:
-            return [self._evaluator.modules[module_name]]
+            module = self._evaluator.modules[module_name]
         except KeyError:
             pass
+        else:
+            return [module] if module else []
```

Comment:
On Windows7 & msys2, i try execute `call jedi#enable_debugging()`
output:
```
speed: init 0.0599999427795
speed: completions start 0.0599999427795
dbg: Parsed D:\Downloads\1.py, with 1 parsers in 1 splits.
speed: parsed 0.0699999332428
warning: No statement under the cursor.
dbg: start: u'datetime' in <fast.FastModule: 1@2-5>
dbg: eval_element <Name: datetime@4,9>@(4, 9)
dbg: finder.filter_name "<Name: datetime@4,9>" in (<ModuleWrapper: <fast.FastModule: 1@2-5>>): u'[<Name: datetime@2,7>]'@(4, 9)
speed: import (<Name: datetime@2,7>,) 2.42000293732
dbg: search_module 'datetime' in 'D:\\Downloads\\1.py'
D:\Portable\msys32\mingw32\lib\python2.7\lib-dynload/datetime.pyd
warning: Module 'D:\\Portable\\msys32\\mingw32\\lib\\python2.7\\lib-dynload/datetime.pyd' not importable.
dbg: after import: [None]
dbg: finder._names_to_types: [<Name: datetime@2,7>] -> [None]
dbg: possible completion scopes: [None]
Traceback (most recent call last):
  File "D:\Portable\gVim\vimfiles\bundle\jedi-vim/jedi_vim.py", line 181, in completions
    completions = script.completions()
  File "D:\Portable\gVim\vimfiles\bundle\jedi-vim/jedi/api/__init__.py", line 192, in completions
    completion_names = get_completions(user_stmt, b)
  File "D:\Portable\gVim\vimfiles\bundle\jedi-vim/jedi/api/__init__.py", line 178, in get_completions
    completion_names += self._simple_complete(path, dot, like)
  File "D:\Portable\gVim\vimfiles\bundle\jedi-vim/jedi/api/__init__.py", line 259, in _simple_complete
    for names_dict in s.names_dicts(search_global=False):
AttributeError: 'NoneType' object has no attribute 'names_dicts'
```
I think it's slashes problem, on `jedi-vim\jedi\evaluate\compiled\__init__.py` L359, dotted_path is `/datetime`